robonomicsinterface/classes/subscriptions.py

```diff
@@ -18,6 +19,12 @@ class SubEvent(Enum):
     NewDevices = "NewDevices"
 
 
+SubscriptionHandler = tp.Union[
+    tp.Callable[[tp.Tuple[str, str, str]], tp.Any],
+    tp.Callable[[tp.Tuple[str, str, str], str], tp.Any],
```

Comment:
What's this `tp.Any` for?

Reply:
All in all, this was simply typed as `callable` due to various options of data types
e.g. For `New Devices` it's a `tp.Tuple[str, tp.List[str]]`

Reply:
> What's this `tp.Any` for?

Can be removed.

Reply:
I'll remove the entire typing notation here due the the enormous amount of types to be mentioned, taking in consideration the newly added event_id which doubles them